demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -288,67 +175,56 @@ def set_content_items(self):
         min_content_items_version = MARKETPLACE_MIN_VERSION
         if content_items:
             min_content_items_version = str(
-                min(parse(content_item.fromversion) for content_item in content_items)
+                min(
+                    parse(content_item.fromversion)
+                    for content_item in content_items
+                    if content_item.content_type
+                    not in (
+                        ContentType.XDRC_TEMPLATE,
+                        ContentType.TRIGGER,
+                        ContentType.TEST_PLAYBOOK,
+                    )
```

Comment:
feels risky, how can we ensure it's maintained? e.g. we soon [might add](https://jira-hq.paloaltonetworks.local/browse/CRTX-89006) a fromversion field to triggers.

Reply:
We can maybe add instead:
```
it content_item.fromversion != "0.0.0"
```

Reply:
ok, use `DEFAULT_CONTENT_ITEM_FROM_VERSION`